source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -345,6 +409,25 @@ S3 Metrics
 
    Total number of S3 requests with 5xx errors.
 
+IAM Metrics
+~~~~~~~~~~~
+
+.. metric:: minio_node_iam_last_sync_duration_millis
+   
+   Last successful IAM data sync duration in milliseconds. 
+
+.. metric:: minio_node_iam_since_last_sync_millis
+   
+   Time (in milliseconds) since last successful IAM data sync. This is set to 0 until the first sync after server start. 
```

Comment:
```suggestion
   Time (in milliseconds) since last successful IAM data sync. 
   
   This value starts at zero and only increments after the first sync after server start.
```

Removing passive.

Reply:
👍